views/networking/load_balancer/overview.erb

```diff
@@ -13,6 +13,23 @@
     ["Stack", @lb.stack.gsub("ip", "IP")],
     ["Load Balancer Port", @lb.src_port],
     ["Application Port", @lb.dst_port],
-    ["HTTP Health Check Endpoint", @lb.health_check_endpoint]
+    ["HTTP Health Check Endpoint", @lb.health_check_endpoint],
+    ["Health Check Protocol", @lb.health_check_protocol.upcase],
+    ["SSL Certificate Status", @lb.cert_enabled_lb? ? (@lb.active_cert&.cert ? "Available" : "Creating") : "SSL Certificates are not enabled for this load balancer"]
```

Comment:
I would only show this if the load balancer is cert enabled, similar like how we should only show the health check endpoint if the load balancer uses http/https (as #3889 does).